test/membrane_http_adaptive_stream/integration_test/sink_bin_integration_test.exs

```diff
@@ -0,0 +1,190 @@
+  @audio_video_tracks_test_path "./test/membrane_http_adaptive_stream/integration_test/tmp/audio_video_tracks/"
+
+  @audio_multiple_video_tracks_sources [
+    {"https://raw.githubusercontent.com/membraneframework/static/gh-pages/samples/big-buck-bunny/bun33s.aac",
+     :AAC, "audio_track"},
+    {"https://raw.githubusercontent.com/membraneframework/static/gh-pages/samples/big-buck-bunny/bun33s_480x270.h264",
+     :H264, "video_480x270"},
+    {"https://raw.githubusercontent.com/membraneframework/static/gh-pages/samples/big-buck-bunny/bun33s_540x360.h264",
+     :H264, "video_540x360"},
+    {"https://raw.githubusercontent.com/membraneframework/static/gh-pages/samples/big-buck-bunny/bun33s_720x480.h264",
+     :H264, "video_720x480"}
+  ]
+  @audio_multiple_video_tracks_ref_path "./test/membrane_http_adaptive_stream/integration_test/fixtures/audio_multiple_video_tracks/"
+  @audio_multiple_video_tracks_test_path "./test/membrane_http_adaptive_stream/integration_test/tmp/audio_multiple_video_tracks/"
+
+  defmodule TestPipeline do
+    use Membrane.Pipeline
+    alias Membrane.HTTPAdaptiveStream
+    alias Membrane.HTTPAdaptiveStream.Storages.FileStorage
+
+    @impl true
+    def handle_init(%{sources: sources, output_dir: output_dir}) do
+      sink_bin = %HTTPAdaptiveStream.SinkBin{
+        muxer_segment_duration: 2 |> Membrane.Time.seconds(),
+        manifest_module: HTTPAdaptiveStream.HLS,
+        target_window_duration: 30 |> Membrane.Time.seconds(),
+        target_segment_duration: 2 |> Membrane.Time.seconds(),
+        persist?: false,
+        storage: %FileStorage{
+          directory: output_dir
+        }
+      }
+
+      children =
+        sources
+        |> Enum.flat_map(fn {source, encoding, track_name} ->
+          parser =
+            case encoding do
+              :H264 ->
+                %Membrane.H264.FFmpeg.Parser{
+                  framerate: {30, 1},
+                  alignment: :au,
+                  attach_nalus?: true
+                }
+
+              :AAC ->
+                %Membrane.AAC.Parser{
+                  out_encapsulation: :none
+                }
+            end
+
+          [
+            {{:source, track_name}, source},
+            {{:parser, track_name}, parser}
+          ]
+        end)
+        |> then(&[{:sink_bin, sink_bin} | &1])
+
+      links =
+        sources
+        |> Enum.map(fn {_source, encoding, track_name} ->
+          link({:source, track_name})
+          |> to({:parser, track_name})
+          |> via_in(Pad.ref(:input, track_name),
+            options: [encoding: encoding, track_name: track_name]
+          )
+          |> to(:sink_bin)
+        end)
+
+      {{:ok, spec: %ParentSpec{children: children, links: links}}, %{}}
+    end
+  end
+
+  describe "Reference HLS content creation should only take place locally." do
+    test "check if fixture creation is unset" do
+      assert not @create_fixtures
```

Comment:
why the hell are you asserting some compile time attribute?

Reply:
Please read the comments from previous reviewers regarding this variable.